### PR TITLE
More Variable Synchronizing

### PR DESCRIPTION
**Pull request by JonathonMisiewicz**
## Description
Three more sets of renamings to synchronize variables across PQE and VQE, which will hopefully be the last bit of cleanup before the work I actually care about.

The review for this can absolutely wait until Monday, but now I won't have this nagging at me all weekend.

## Checklist
- [x] Ready to go!

### CODE REVIEW

File: src/qforte/abc/algorithm.py [hunk 1]
```diff
@@ -56,6 +56,11 @@ class Algorithm(ABC):
         The number of pauli terms (Hermitian products of Pauli X, Y, and/or Z gates)
         mesaured over the entire algorithm.
 
+    _res_vec_evals : int
+        The total number of times the entire residual was evaluated.
+
+    _res_m_evals : int
+        The total number of times an individal residual element was evaluated.
```
Comment (imagoulas) [inline src/qforte/abc/algorithm.py:63]: "individal" -> "individual"

File: src/qforte/abc/algorithm.py [hunk 2]
```diff
@@ -199,6 +204,11 @@ class AnsatzAlgorithm(Algorithm):
     _opt_maxiter : int
         The maximum number of iterations for the classical optimizer.
 
+    _opt_thresh : float
+        The numerical convergence threshold for the specified classical
+        optimization algorithm. Is usually the norm of the gradient, but
+        is algorithm dependant, see scipy.minimize.optimize for detials.
```
Comment (imagoulas) [inline src/qforte/abc/algorithm.py:210]: "detials" => "details"

**Review by imagoulas — APPROVED** [review]
Everything looks good to me. I added a couple of comments regarding typos in the comments to the source code (I noticed that in earlier PRs you were also fixing those).

**Comment by JonathonMisiewicz** [comment]
Thanks for the catches, Ilias. Nick left a lot of them (that's what happens when you're coding alone), so every one we can catch helps.

**Pull request merged**